pkg/kubectl/cmd/cp/cp.go

```diff
@@ -280,6 +280,15 @@ func (o *CopyOptions) copyFromPod(src, dest fileSpec) error {
 		return errFileCannotBeEmpty
 	}
 
+	stat, err := os.Stat(dest.File)
```

Comment:
Thanks!

/ok-to-test
/priority backlog

Reply:
Thanks a lot for taking the time to review!